File: app/src/main/java/com/example/elsafa/ui/account/AccountFragment.java
package com.example.elsafa.ui.account;

import android.content.Intent;
import android.os.Bundle;
import android.transition.TransitionInflater;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.FrameLayout;
import android.widget.RadioButton;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import com.example.elsafa.CafeLocation;
import com.example.elsafa.R;

public class AccountFragment extends Fragment {


    private FrameLayout linear_layout_map;
    private RadioButton black_theme;

    public View onCreateView(@NonNull LayoutInflater inflater,
                             ViewGroup container, Bundle savedInstanceState) {
        View root = inflater.inflate(R.layout.fragment_account, container, false);

        TransitionInflater inflater1 = TransitionInflater.from(getContext());
        setExitTransition(inflater1.inflateTransition(R.transition.fragment_in));
        setEnterTransition(inflater1.inflateTransition(R.transition.fragment_out));


        black_theme = root.findViewById(R.id.black_theme);
        linear_layout_map = root.findViewById(R.id.linear_layout_map);

        setListeners();

        return root;
    }

    private void setListeners() {
        setMapListener();
    }

    private void setMapListener() {
        linear_layout_map.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getContext(), CafeLocation.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/whiteside/cafe/Account/AccountFragment.kt
package com.whiteside.cafe.Account

import android.content.Intent
import android.os.Bundle
import android.transition.TransitionInflater
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.FrameLayout
import androidx.fragment.app.Fragment
import com.whiteside.cafe.CafeLocation
import com.whiteside.cafe.R

class AccountFragment : Fragment() {
    private var linear_layout_map: FrameLayout? = null
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?, savedInstanceState: Bundle?
    ): View? {
        val root = inflater.inflate(R.layout.fragment_account, container, false)
        val inflater1 = TransitionInflater.from(context)
        exitTransition = inflater1.inflateTransition(R.transition.fragment_in)
        enterTransition = inflater1.inflateTransition(R.transition.fragment_out)
        linear_layout_map = root.findViewById(R.id.linear_layout_map)
        setListeners()
        return root
    }

    private fun setListeners() {
        setMapListener()
    }

    private fun setMapListener() {
        linear_layout_map.setOnClickListener(View.OnClickListener {
            val intent = Intent(context, CafeLocation::class.java)
            startActivity(intent)
        })
    }
}